demo

```diff
@@ -1,12 +1,32 @@
 #!/bin/bash
 
-
 CLIENT="-f docker-compose.client.yml"
 if [ -n "${NO_CLIENT}" ]; then
     unset CLIENT
     echo "-- disabling client containers"
 fi
 
+if [ "$#" -eq 0 ]; then
+    echo "Arguments passed to this command are passed directly to docker-compose, if you want to run the demo, run:"
+    echo ""
+    echo "'./demo up'"
+    exit 1
+fi
+
+# speed up first start by pulling containers in parallel
+if [ ! -e ".first-run" ]; then
```

Comment:
I think a better check is whether the images already exist locally. It should be enough to check one of them.

Reply:
Ok, updated!